lib/qstash/version.rb

```diff
@@ -1,3 +1,3 @@
 module QStash
-  VERSION = "0.1.0"
+  VERSION = "0.1.1"
```

Comment:
Hope you publish a new version of this gem

Reply:
Please remove - I will update in a separate PR if needed